repository_service_tuf/cli/admin/sign.py

```diff
@@ -24,13 +26,53 @@
     _filter_root_verification_results,
     _print_keys_for_signing,
     _print_root,
-    _warn_no_save,
 )
+from repository_service_tuf.helpers.api_client import (
+    URL,
+    Methods,
+    request_server,
+    send_payload,
+    task_status,
+)
+
+
+def _get_pending_roles(
+    settings: Any, api_server: Optional[str] = None
+) -> Dict[str, Dict[str, Any]]:
+    if api_server:
+        settings.SERVER = api_server
+
+    if settings.get("SERVER") is None:
+        api_server = Prompt.ask("\n[cyan]API[/] URL address")
+        settings.SERVER = api_server
```

Comment:
I am okay to drop the settings check, but there is something I am missing as a usage for `settings.SERVER`.
I see we set it in other commands as well such as `add artifacts`, `delete artifacts` and `import-artifacts`.
@kairoaraujo if we set it in one of the commands will it be setup for my next command?

Reply:
What is missing in `settings.SERVER` is the documentation for the user.
As a practical example, I utilize my `~/.rstuf.yml` with `SERVER` defined, which empowers me to effectively not fill the server or address by the prompt every time. 
(BTW, @MVrachev, your code works with it, I had to comment to test it)
```shell
                                                                 
❯ rstuf admin metadata sign


┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃                                Metadata Signing Tool                                ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
╭─ Error ─────────────────────────────────────────────────────────────────────────────╮
│ {"detail":{"message":"No metadata pending signing available","error":"Requires      │
│ bootstrap started. State: None"}}                                                   │
╰─────────────────────────────────────────────────────────────────────────────────────╯
                                                                                       
```

                 
The problem here is that we focus on the code and we forgot to document it.
For example, if you setup in the `rstuf.yaml` 

Regarding arg and prompt, prompt is better as it is interactive.

Reply:
> Regarding arg and prompt, prompt is better as it is interactive

IMO interactivity isn't better per se, and we did have the discussion before, about which inputs should be provided via cli arg, and which via prompt. From what I remember ,you can argue both ways.

pro all prompt: consistent "wizard" ux for all inputs
pro cli args and prompt: inputs like server address, input file path and output file path differ are different types of  inputs compared to the rest (keys, threshold, signers etc.) The former can be seen as config parameters to even use the tool, and the latter are what the tool is all about.

I lean towards using cli args for the config inputs, because they are generally easier to develop/maintain and to use. Independently, it's a nice additional feature to support providing these args via config file.

Reply:
I think I missed/forgot this discussion.
We should document it in our `docs/devel/design.rst`

Reply:
I think I support @kairoaraujo more on this one as for regular users of rstuf CLI it makes sense to have it configured.
If we want to continue this discussion I don't mind, but I don't feel this pr is the place to do it.
I will add `settings.SERVER` as an option input for now as that's how we have proceeded until now.
If we agree on something else we need to backtrack, not only for this function but everywhere to be consistent.